gulpfile.js

```diff
@@ -1,201 +1,196 @@
-    .pipe(include())
-    .pipe(babel({
-        presets: [["env", {
-          "targets": {
-            "browsers": ["last 4 versions"]
-        },
-        loose: true,
-        modules: false,
-        }]
-        ]
+        .pipe(insert.prepend(version + "\n"))
+        .pipe(include())
+        .pipe(babel({
+            presets: [["env", {
+                "targets": {
+                    "browsers": ["last 4 versions"]
+                },
+                loose: true,
+                modules: false,
+            }]
+            ]
         }))
-    .pipe(concat('tabulator_core.js'))
-    .pipe(gulp.dest('dist/js'))
-    .pipe(rename({suffix: '.min'}))
-    .pipe(uglify())
-    .pipe(insert.prepend(version))
-    .pipe(gulp.dest('dist/js'))
-    .on('end', function(){ gutil.log('Core complete'); })
-    });
+        .pipe(concat('tabulator_core.js'))
+        .pipe(gulp.dest('dist/js'))
+        .pipe(rename({ suffix: '.min' }))
+        .pipe(uglify())
+        .pipe(insert.prepend(version))
+        .pipe(gulp.dest('dist/js'))
+        .on('end', function () { gutil.log('Core complete'); })
+});
 
 
 
 //make jquery wrapper
-gulp.task('modules', function(){
+gulp.task('modules', function (done) {
 
     var path = __dirname + "/src/js/modules/";
 
     var files = fs.readdirSync(path);
 
     var core = ["layout.js", "localize.js", "comms.js"];
 
-    files.forEach(function(file, index){
+    files.forEach(function (file, index) {
 
-        if(!core.includes(file)){
+        if (!core.includes(file)) {
             return gulp.src('src/js/modules/' + file)
-            .pipe(insert.prepend(version + "\n"))
-            .pipe(include())
-            .pipe(babel({
-                presets: [["env", {
-                  "targets": {
-                    "browsers": ["last 4 versions"]
-                },
-                loose: true,
-                modules: false,
-                }]
-                ]
+                .pipe(insert.prepend(version + "\n"))
+                .pipe(include())
+                .pipe(babel({
+                    presets: [["env", {
+                        "targets": {
+                            "browsers": ["last 4 versions"]
+                        },
+                        loose: true,
+                        modules: false,
+                    }]
+                    ]
                 }))
-            .pipe(concat(file))
-            .pipe(gulp.dest('dist/js/modules/'))
-            .pipe(rename({suffix: '.min'}))
-            .pipe(uglify())
-            .pipe(insert.prepend(version))
-            .pipe(gulp.dest('dist/js/modules/'))
+                .pipe(concat(file))
+                .pipe(gulp.dest('dist/js/modules/'))
+                .pipe(rename({ suffix: '.min' }))
+                .pipe(uglify())
+                .pipe(insert.prepend(version))
+                .pipe(gulp.dest('dist/js/modules/'))
         }
-        });
-
     });
 
+    gutil.log("Modules complete");
```

Comment:
Added notification when modules was complete. Called done function to let gulp know when the async function was done